test/functional/normal/jump_spec.lua

```diff
@@ -1,4 +1,7 @@
 local helpers = require('test.functional.helpers')(after_each)
+local clear, feed = helpers.clear, helpers.feed
+local redir_exec = helpers.redir_exec
+local eq = helpers.eq
```

Comment:
linter is failing here because many of these are redundant (see lines 6, 8, etc.)

```
Checking test/functional/normal/jump_spec.lua     [0m[31m[1m6 warnings[0m

    test/functional/normal/jump_spec.lua:2:7: unused variable [0m[1mclear[0m
    test/functional/normal/jump_spec.lua:2:14: unused variable [0m[1mfeed[0m
    test/functional/normal/jump_spec.lua:4:7: unused variable [0m[1meq[0m
    test/functional/normal/jump_spec.lua:6:7: variable [0m[1mclear[0m was previously defined on line 2
    test/functional/normal/jump_spec.lua:8:7: variable [0m[1meq[0m was previously defined on line 4
    test/functional/normal/jump_spec.lua:10:7: variable [0m[1mfeed[0m was previously defined on line 2
```

Reply:
Fixed in https://github.com/neovim/neovim/pull/11530/commits/0c59972ae46d67a981f26f2f933c7ec926122c32 .